app/models/miq_provision_virt_workflow.rb

```diff
@@ -157,6 +157,10 @@ def validate_vm_name(field, values, dlg, fld, value)
     validate_length(field, values, dlg, fld, value)
   end
 
+  def validate_email(field, values, dlg, fld, value)
```

Comment:
@syncrou This method is not needed, the dialogs should call `validate_regex` directly.